### DISCUSSION TITLE
Breadcrumbs on routes with _suffix

### DISCUSSION THREAD

**Discussion by pooryamoh** [TanStack/router#1763] [Q&A] [answered]
Hi,

I want to create breadcrumb using `useMatches()`, I want my links looks nested but I don't want my components to be nested, I tried to use routes with _suffix, but the problem is that `useMatches()` won't have matches related to parent route for example, I have links `posts/1/edit`, and I want different non nested components without `<Outlet/>` on all `posts`, `posts/1` and `posts/1/edit` routes, while have proper bread crumb on each page.Also using index on each page will add two matches. How do you implement something?

**Comment by schiller-manuel** [comment]
can you please show a minimum complete example by forking one of the existing examples on stackblitz?

**Reply by pooryamoh** [reply, marked as answer]
@schiller-manuel I manged to get what I want using solution in discussion #895.
